include/grpcpp/security/auth_metadata_processor.h

```diff
@@ -27,29 +27,39 @@
 
 namespace grpc {
 
-/// Interface allowing custom server-side authorization based on credentials
-/// encoded in metadata.  Objects of this type can be passed to
-/// \a ServerCredentials::SetAuthMetadataProcessor().
+// Interface allowing custom server-side authorization based on credentials
```

Comment:
Please use three slashes instead of two for all of these comments, so that the comments show up in the doxygen docs.